src/api.js

```diff
@@ -0,0 +1,49 @@
+const BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const STORAGE_KEY = 'appId';
+
+const post = (endpoint, body = {}, method = 'POST') => fetch(`${BASE_URL}${endpoint}`, {
+  method,
+  body: JSON.stringify(body),
+  headers: {
+    'Content-type': 'application/json; charset=UTF-8',
+  },
+});
+
+export const createApp = async () => {
+  let appId = localStorage.getItem(STORAGE_KEY);
+  if (!appId) {
+    const response = await post('/apps');
+
+    appId = await response.text();
+    localStorage.setItem(STORAGE_KEY, appId);
+  }
+};
```

Comment:
- [ ] Your Thunk action creator should remove a book from the API, not the local storage, kindly implement fetch or Axios for this functionality so your app removes a book from the API